bindings/csharp/Keystone.Net/Keystone.Net.csproj

```diff
@@ -1,28 +1,36 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.1</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
     <RootNamespace>Keystone</RootNamespace>
 
-    <Version>1.1.0</Version>
-    <AssemblyVersion>$(Version)</AssemblyVersion>
-    <FileVersion>$(Version).0</FileVersion>
+    <Version>0.9.1.1</Version>
+    <AssemblyVersion>0.9.1.1</AssemblyVersion>
+    <FileVersion>0.9.1.1</FileVersion>
 
-    <Description>.NET bindings to the Keystone Engine.</Description>
-    <Authors>Grégoire Geis</Authors>
+    <Description>Keystone is a lightweight multi-platform, multi-architecture assembler framework. This package corresponds to the csharp bindings in the official git repository.</Description>
+    <Authors>Nguyen Anh Quynh, Marco Fornaro, Grégoire Geis, Jämes Ménétrey</Authors>
 
-    <PackageId>Keystone.Net</PackageId>
+    <PackageId>keystoneengine.csharp</PackageId>
```

Comment:
Why renaming to `KeystoneEngine.CSharp`? Usually packages that end with `.CSharp` are related to the language itself (ie. `Microsoft.CodeAnalysis.CSharp`). Bindings are available to all of .NET (not only C#) and thus usually end with .NET.

However this does pose a problem: How do we decrease the current version? (I'm not sure NuGet allows this).